shell/platform/BUILD.gn

```diff
@@ -12,7 +12,9 @@ group("platform") {
       "android",
     ]
   } else if (is_linux) {
-    deps = []
+    deps = [
+      "linux",
```

Comment:
Though we may eventually want this, you're going to end up making a lot of bots mad because of unresolved build dependencies. I'd suggest making this opt-in and selectively enabling this on the bots. Your call but this will probably ease the transition.

Reply:
Removed deps on all the new targets for now, since I'm still blocked on the setup script.